test/helpers/utils.js

```diff
@@ -36,6 +36,12 @@ window.Utils = {
 
     getPointsFromPath: function(pathData) {
         return pathData.split(/[,ML]/).map(Number);
+    },
+
+    zoom: function(chartEl, coords) {
```

Comment:
I was a little surprised the argument was a pair of coordinates. With a function named `zoom`, I'd tend to expect a zoom level. Would this be more accurately named `zoomToFit`, `zoomToRegion`, or something like that?